internal/buildscripts/packaging/installer/install.sh

```diff
@@ -1196,17 +1195,33 @@ parse_args_and_install() {
       --without-systemd-instrumentation)
         with_systemd_instrumentation="false"
         ;;
-      --with-java-instrumentation)
-        with_java_instrumentation="true"
-        ;;
-      --without-java-instrumentation)
+      --with-instrumentation-sdk)
         with_java_instrumentation="false"
-        ;;
-      --with-node-instrumentation)
-        with_node_instrumentation="true"
-        ;;
-      --without-node-instrumentation)
         with_node_instrumentation="false"
+        echo "$2" | tr ',' '\n' | while read lang; do
```

Comment:
```suggestion
        for lang in $(echo "$2" | tr ',' ' '); do
```
Otherwise the local var assignments below will be lost (see https://www.shellcheck.net/wiki/SC2031).